CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#7270](https://github.com/osmosis-labs/osmosis/pull/7270) feat: eip target gas from consensus params
 
 ### Bug Fixes
+* [#7360](https://github.com/osmosis-labs/osmosis/pull/7360) fix: use gov type for SetScalingFactorController
```

Comment:
should be in the unreleased section 🙏

Reply:
https://github.com/osmosis-labs/osmosis/pull/7360/commits/f523035a376fa97aef1f9d41e445f61ef3eae37f